projects/kit/components/input-count/input-count-options.ts

```diff
@@ -0,0 +1,42 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiSizeL} from '@taiga-ui/core';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface InputCountOptions {
+    readonly icons: Readonly<{
+        up: PolymorpheusContent<TuiContextWithImplicit<TuiSizeL>>;
```

Comment:
Buttons are rather tiny, they always have size "s". How about we simplify it by keeping context `{}` — this way you can just pass it to buttons' `icon` input, as it is already polymorpheus.

Reply:
But buttons' height gets smaller when input-count uses "s" size. And this polymorpheus context is taken into account textfield size

Reply:
I meant that you probably cannot fit a big icon anyway, even for a big input size. Template complexity really increases now and I don't want this token option feature to become an overkill.

Reply:
Ok. I agree about template complexity. I've changed it to {} context

Reply:
@waterplea there is also tuiTableMode directive with TUI_TEXTFIELD_APPEARANCE token. Should we add TUI_INPUT_COUNT_OPTIONS here as well? In this case should we move TUI_INPUT_COUNT_OPTIONS into core package?